### DISCUSSION TITLE
Carousel within a Daisy Responsive card?

### DISCUSSION THREAD

**Discussion by sarahk** [saadeghi/daisyui#691] [Q&A] [answered]
I'm new to tailwind and daisy so probably missing something obvious. 

I want to use https://daisyui.com/components/card/ but replace the `<figure>` and `<img>` with a horizontally scrolling carousel with either dots or arrows for navigations or with the images at w-3/4

Does anyone have an example that I could take a look at?

**Comment by saadeghi** [comment, marked as answer]
Here's an example:  
https://stackblitz.com/edit/daisyui-svelte-vite-7aukeb?file=src/App.svelte

I added `md:h-full md:w-auto` to `carousel` because when card is horizontal we want the carousel to fill the height.  
And I made dots `absolute` because it looks better on the image.

**Reply by sarahk** [reply]
Oh you make that look so easy. I've just managed to make it work in seconds which is amazing after the hours I've spent trying to get other carousel methods to work. Thank you!!!!
